src/SuperDump.Analyzer.Linux/Analysis/CoreDumpAnalyzer.cs

```diff
@@ -173,5 +183,30 @@ public class CoreDumpAnalyzer {
 		private IFileInfo FindCoredumpOrNull(IDirectoryInfo directory) {
 			return directory.EnumerateFiles("*.core", SearchOption.AllDirectories).FirstOrDefault();
 		}
+
+		private string RunBashCommand(string command) {
+			string escapedCommand = command.Replace("\"", "\\\"");
+			var process = new Process {
```

Comment:
Use `ProcessRunner` from SuperDump.Common here. It already takes care of redireting StdOut+StdErr into strings.